### PR TITLE
Update tune-performance.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tune-performance.md [hunk 1]
```diff
@@ -29,6 +29,24 @@ The results are displayed in the form of a table, and you can sort the results b
 
 For details, see [Statement Execution Details in TiDB Dashboard](https://docs.pingcap.com/tidb/stable/dashboard-statement-details).
 
+## Slow Queries
+
+To use the Slow Queries, perform the following steps:
```
Comment (qiancai) [inline tidb-cloud/tune-performance.md:34]:
```suggestion
By default, SQL queries that take more than 300 milliseconds are considered as slow queries. 

To view slow queries in a cluster, perform the following steps:
```

File: tidb-cloud/tune-performance.md [hunk 2]
```diff
@@ -29,6 +29,24 @@ The results are displayed in the form of a table, and you can sort the results b
...
+
+1. Navigate to the **Diagnosis** tab of a cluster.
+
+2. Click the **Slow Queries** tab.
+
+3. Click any item in the list to display detailed execution information of the slow query.
```
Comment (qiancai) [inline tidb-cloud/tune-performance.md:40]:
```suggestion
3. Click any slow query in the list to display its detailed execution information.
```

File: tidb-cloud/tune-performance.md [hunk 3]
```diff
@@ -29,6 +29,24 @@ The results are displayed in the form of a table, and you can sort the results b
...
+
+4. (Optional) If you care about specific time periods, databases, keywords, you can select the corresponding schema(s) to filter the results.
```
Comment (qiancai) [inline tidb-cloud/tune-performance.md:42]:
```suggestion
4. (Optional) You can filter slow queries based on the target time range, the related database, SQL keywords, SQL types, and limit the number of slow queries to be displayed.
```

**Comment by qiancai** [comment]
The info of Slow Queries also needs to be added in L8 ~ L12.

File: TOC-tidb-cloud.md [hunk 4]
```diff
@@ -123,6 +123,7 @@
   - [Overview](/tidb-cloud/tidb-cloud-tune-performance-overview.md)
   - Analyze Performance
     - [Statement Analysis](/tidb-cloud/tune-performance.md)
+    - [Slow Queries](/tidb-cloud/tune-performance.md#slow-queries)
     - [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer)
     - [Statement Summary Tables](/statement-summary-tables.md)
```
Comment (qiancai) [inline TOC-tidb-cloud.md:128]:
```suggestion
    - [Use the Diagnosis Tab](/tidb-cloud/tune-performance.md)
    - [Use Statement Summary Tables](/statement-summary-tables.md)
```

Comment (qiancai) [reply TOC-tidb-cloud.md:128]: Currently, the docs website does not support using subsections in the TOC file any more. There is a CI error reported.

File: tidb-cloud/tune-performance.md [hunk 5]
```diff
@@ -29,6 +29,24 @@ The results are displayed in the form of a table, and you can sort the results b
...
+
+The results are displayed in the form of a table, and you can sort the results by different columns.
+
+![Slow Queries]()
```
Comment (qiancai) [inline tidb-cloud/tune-performance.md:46]: Please add the screenshot.

Comment (lilyjazz) [reply tidb-cloud/tune-performance.md]: done

**Comment by lilyjazz** [comment]
@baurine PTAL

**Review by baurine — APPROVED** [review]

File: tidb-cloud/tune-performance.md [hunk 6]
```diff
@@ -29,6 +29,26 @@ The results are displayed in the form of a table, and you can sort the results b
...
+
+To view slow queries in a cluster, perform the following steps:
+
+1. Navigate to the **Diagnosis** tab of a cluster.
+
+2. Click the **Slow Queries** tab.
+
+3. Click any slow query in the list to display its detailed execution information.
+
+4. (Optional) You can filter slow queries based on the target time range, the related database, SQL keywords, SQL types, and limit the number of slow queries to be displayed.
```
Comment (qiancai) [inline tidb-cloud/tune-performance.md:44]:
```suggestion
4. (Optional) You can filter slow queries based on the target time range, the related database, and SQL keywords, and also limit the number of slow queries to be displayed.
```

Comment (qiancai) [reply tidb-cloud/tune-performance.md:44]: It seems that SQL type is not provided in the filter conditions.

File: tidb-cloud/tune-performance.md [hunk 7]
```diff
@@ -29,6 +29,26 @@ The results are displayed in the form of a table, and you can sort the results b
...
+
+3. Click any slow query in the list to display its detailed execution information.
+
+4. (Optional) You can filter slow queries based on the target time range, the related database, and SQL keywords, and also limit the number of slow queries to be displayed.
+
+The results are displayed in the form of a table, and you can sort the results by different columns.
+
+![Slow Queries](/media/tidb-cloud/slow-queries.png)
+
+For more information, see [Slow Queries in TiDB Dashboard](https://docs.pingcap.com/tidb/dev/dashboard-slow-query).
```
Comment (qiancai) [inline tidb-cloud/tune-performance.md:50]:
```suggestion
For more information, see [Slow Queries in TiDB Dashboard](https://docs.pingcap.com/tidb/stable/dashboard-slow-query).
```

File: tidb-cloud/tune-performance.md [hunk 8]
```diff
@@ -29,6 +29,26 @@ The results are displayed in the form of a table, and you can sort the results b
 
 For details, see [Statement Execution Details in TiDB Dashboard](https://docs.pingcap.com/tidb/stable/dashboard-statement-details).
```
Comment (qiancai) [inline tidb-cloud/tune-performance.md:30]:
```suggestion
For more information, see [Statement Execution Details in TiDB Dashboard](https://docs.pingcap.com/tidb/stable/dashboard-statement-details).
```

File: tidb-cloud/tune-performance.md [hunk 9]
```diff
@@ -5,10 +5,12 @@ summary: Learn how to analyze and tune performance of your TiDB Cloud cluster.
 
 # Analyze and Tune Performance
 
-TiDB Cloud provides [Statement Analysis](#statement-analysis) and [Key Visualizer](#key-visualizer) to analyze performance.
+TiDB Cloud provides [Statement Analysis](#statement-analysis), [Slow Query](#slow-query) and [Key Visualizer](#key-visualizer) to analyze performance.
```
Comment (qiancai) [inline tidb-cloud/tune-performance.md:8]:
```suggestion
TiDB Cloud provides [Statement Analysis](#statement-analysis), [Slow Query](#slow-query), and [Key Visualizer](#key-visualizer) to analyze performance.
```

File: tidb-cloud/tune-performance.md [hunk 10]
```diff
@@ -27,7 +29,27 @@ The results are displayed in the form of a table, and you can sort the results b
...
+
+To view slow queries in a cluster, perform the following steps:
+
+1. Navigate to the **Diagnosis** tab of a cluster.
+
+2. Click the **Slow Query** tab.
+
+3. Click any slow query in the list to display its detailed execution information.
+
+4. (Optional) You can filter slow queries based on the target time range, the related database, and SQL keywords, and also limit the number of slow queries to be displayed.
```
Comment (TomShawn) [inline tidb-cloud/tune-performance.md:46]:
```suggestion
4. (Optional) You can filter slow queries based on the target time range, the related databases, and SQL keywords. You can also limit the number of slow queries to be displayed.
```

File: TOC-tidb-cloud.md [hunk 11]
```diff
@@ -122,9 +122,8 @@
 - Tune Performance
   - [Overview](/tidb-cloud/tidb-cloud-tune-performance-overview.md)
   - Analyze Performance
-    - [Statement Analysis](/tidb-cloud/tune-performance.md)
-    - [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer)
```
Comment (TomShawn) [inline TOC-tidb-cloud.md:126]: Is Key Visualizer a lesser used feature?

**Review by TomShawn — APPROVED** [review]
minor comments
rest LGTM

File: TOC-tidb-cloud.md (hunk 11 above)
Comment (qiancai) [reply TOC-tidb-cloud.md:126]: image

**Review by qiancai — APPROVED** [review]

**Pull request merged**